app/Resources/views/layout/scripts.html.twig

```diff
@@ -0,0 +1,5 @@
+{% javascripts
+    'assets/js/app.js'
+%}
+<script type="text/javascript" src="{{ asset_url }}"></script>
```

Comment:
It's consistent with : 

https://github.com/ezsystems/ezplatform-demo/blob/master/app/Resources/views/layout/styles.html.twig
https://github.com/ezsystems/ezplatform-demo/blob/master/app/Resources/views/layout/scripts.html.twig